docking.js

```diff
@@ -488,11 +489,17 @@ var DockedDash = GObject.registerClass({
         ], [
             settings,
             'changed::dash-max-icon-size',
-            () => { this.dash.setIconSize(settings.get_int('dash-max-icon-size')); }
+            () => {
+                this.dash.setIconSize(settings.get_int('dash-max-icon-size'));
+                this._updateVisibleDesktop();
```

Comment:
This change is not needed as will be triggered by a reallocation anyways.

Reply:
I, personally, would prefer to keep them, because I always have preferred being explicit than implicit. Also, that was precisely another reason for the timeout when sending the data: to coalesce all these changes in a single one. But, of course, if you consider that they can be safely remove, then we can remove them.